docs/api-npmscripts/lintfiles.md

```diff
@@ -0,0 +1,29 @@
+# lint [files..]
```

Comment:
Is this the same file as above?

Reply:
This is the same content, but the name of the file has changed to be parseable.